code/modules/research/techweb/all_nodes.dm

```diff
@@ -1523,6 +1529,12 @@
 		"c38_hotshot",
 		"c38_iceblox",
 		"techshotshell",
+		// SKYRAT EDIT BEGIN - replace this when i figure out a good way to fix the modular method
+		"security_pistol_magazine_iceblox",
+		"security_pistol_magazine_cruelty",
```

Comment:
dont forget to change the other mentions of cruelty/influence now that the names are different